auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/RestWrapper.java

```diff
@@ -83,26 +83,22 @@ public void setRecord(Object record) {
 
     @JsonProperty("TotalRecordCount")
     public int getTotalRecordCount() {
-        if (records != null) {
-            if (records instanceof Collection) {
+
+            if ( records != null && records instanceof Collection) {
                 Collection c = (Collection) records;
                 for (Object obj : c) {
                     try {
                         Method method = obj.getClass().getMethod("getCounter");
-                        Integer count = (Integer) method.invoke(obj);
-                        return count;
-                    } catch (NoSuchMethodException e) {
-                        return 0;
-                    } catch (InvocationTargetException e) {
-                        return 0;
-                    } catch (IllegalAccessException e) {
+                        return (Integer) method.invoke(obj);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "count". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1488)